.scrutinizer.yml

```diff
@@ -0,0 +1,27 @@
+checks:
+    python:
+        code_rating: true
+        duplicate_code: true
+build:
+    environment:
+        python: 3.11.8
+        postgresql: false
+        redis: false
+    dependencies:
+        before:
+            - pip install tox coverage
```

Comment:
The intention to have `.scrutinizer.yml` is good but it can generate way more code maintenance for us, for instance if we need to pin `tox` again or if when we upgrade python again that would require extra PRs. Currently we're relying on a global config from scrutinizer that applies to all repos (and we'll update it accordingly before merging the PRs):

```
checks:
    python:
        code_rating: true
        duplicate_code: true
filter:
    paths: ['*']
build:
    environment:
        python: 3.9.12
        postgresql: false
        redis: false
    dependencies:
        before:
            - pip install tox==3.27.1 virtualenv==20.21.0 coverage
    tests:
        override:
            -
                command: 'tox'
                idle_timeout: 300
                coverage:
                    file: '.coverage'
                    config_file: '.coveragerc'
                    format: 'py-cc'
    nodes:
        analysis:
            tests:
                override:
                    - py-scrutinizer-run
        tests: true
```      

That said, I recommend removing this `.scrutinizer.yml` here.

Reply:
Removed